lib/bundles/inspec-compliance/api/login.rb

```diff
@@ -0,0 +1,134 @@
+# encodeing: utf-8
+# author: Christoph Hartmann
+# author: Dominik Ricter
+# author: Jerry Aldrich
+
+module Login
+  def login(options)
+    options['server_type'] = Compliance::API.determine_server_type(options['server'], options['insecure'])
+
+    return Login::ComplianceServer.login(options) if options['server_type'] == 'compliance'
+    verify_thor_options(options)
+
+    options['url'] = options['server'] + '/compliance'
+    token = options['dctoken'] || options['token']
+    puts store_access_token(options, token)
```

Comment:
I know it's a WIP, to just a reminder to remove this :)

Reply:
`store_access_token` returns a string at this time. I will look at the flow and see if we should pass this up as a return from `login`.

Will investigate.